extensions/amp-autocomplete/0.1/test/validator-amp-autocomplete.out

```diff
@@ -236,6 +236,24 @@ FAIL
 |        </template>
 |      </amp-autocomplete> 
 |
+|      <!-- Valid: amp-component with remote data and rich content template -->
```

Comment:
I believe this is duplicated. (Was added originally in https://github.com/ampproject/amphtml/pull/22094/files#diff-d24e9826474d8ed4abd593084adc2b79; `patch` got confused and added it again.)

Reply:
Nevermind, subsequent commit.